test/integration/install_test.go

```diff
@@ -842,8 +930,10 @@ func TestLogs(t *testing.T) {
 }
 
 func TestEvents(t *testing.T) {
-	for _, err := range testutil.FetchAndCheckEvents(TestHelper) {
-		testutil.AnnotatedError(t, "Error checking events", err)
+	if !TestHelper.Calico() {
+		for _, err := range testutil.FetchAndCheckEvents(TestHelper) {
+			testutil.AnnotatedError(t, "Error checking events", err)
+		}
```

Comment:
Unknown events appearing because of Calico?

Reply:
Yep, which makes the tests fail.

Reply:
Can you please add a comment there explaining these  warnings are just Calico converging to a correct state so they're safe to ignore ?(assuming that's what those messages are about)

Reply:
Looks like `TestEvents` are not necessary after we fixed the installation order of `Calico` and `LinkerdCNI`. Thank you for the suggestion :+1: